test/l4lb/test.sh

```diff
@@ -135,11 +135,11 @@ docker exec -t lb-node docker exec -t cilium-lb \
 
 # Do not stop on error
 set +e
-# Issue 10 requests to LB (with 500ms timeout) which are expected to timeout
+# Issue 10 requests to LB, we expect all to fail due to a Failed connection
 for i in $(seq 1 10); do
     curl -o /dev/null -m 0.5 "${LB_VIP}:80"
-    # code 28 - Operation timeout
-    if [ ! "$?" -eq 28 ]; then
+    # code 7 - Failed to connect ... : No route to host
```

Comment:
Yes it will. I believe we expect the code to compile between every commit, not for the full CI suite to pass for every commit. That is what https://github.com/cilium/cilium/actions/runs/6197972695/job/16827550943 checks at least.